src/providers/twitch/api/Helix.hpp

```diff
@@ -1361,7 +1361,9 @@ class Helix final : public IHelix
         FailureCallback<HelixGetModeratorsError, QString> failureCallback);
 
 private:
-    NetworkRequest makeRequest(QString url, QUrlQuery urlQuery);
+    NetworkRequest makeRequest(
```

Comment:
I would rather see us be explicit in the method with separate functions, e.g. `makeGet`, `makePost`, `makeDelete`, `makePatch` if we want to refactor this. Otherwise, this doesn't do much than move the parameter from the builder to the function itself